README.md

```diff
@@ -6,6 +6,16 @@
 
 ### Data Helpers
 
+Target, extract, and modify structured data from ArchiveSpace to allow users to export, update, add, and delete data. Data helpers leverage the abstraction layer of ArchivesSnake to get and post changes back to a JSONModel object(s) either in bulk, or by specifying a unique identifier. Data helpers enable cleanup tasks, changes to conform to new organizational policies and procedures, and to validate against existing archival and institutional standards. Data helper functionality includes:
+
+- Retrieve and modify notes attached to archival objects or resources
+- Retrieve and modify location data
+- Retrieve and modify container data
+- Identify unlinked agents, subjects, top containers, and digital objects
+- Identify and retrieve undated archival objects
```

Comment:
Why are you just identify and retrieving these and not modifying like in other sections?

Reply:
I believe we were trying to refer to getting a list of undated archival objects (line 15), but as I type that, it seems that should go under serializing (?)

Reply:
There's some overlap in deleting too, since we have a section for deleting unlinked objects, so I'm not sure where those best fit. I think you could maybe condense undated into `Retrieve and modify date data`

Reply:
@p-galligan do you mean just condense lines 15 and 16 into 'Retrieve and modify date data'? Cool with that - I think this gets to the question of how specific we need to get here. I'm cool to leave things general (as we have overall, I think).

Reply:
Yeah, I think that's what I'm generally trying to get at. That we should just be able to cover _all_ date data issues, and we can generalize that in the same way.